packages/ckeditor5-engine/src/model/rootelement.ts

```diff
@@ -30,6 +30,13 @@ export default class RootElement extends Element {
 	 */
 	public _isAttached = true;
 
+	/**
+	 * Informs if the root element is loaded (default).
+	 *
+	 * @internal
+	 */
+	public _isLoaded = true;
```

Comment:
I'm just wondering if this flag is needed. Wouldn't `_isAttached` be enough?

Reply:
I initially thought that maybe one flag would be enough, but these are different states.

*   `_isAttached` -> is it still a part of the document?
*   `_isLoaded` -> is it managed by the current instance of the editor?

The difference makes more sense if you factor features like RTC or revision history. Also, I believe it is better to have two separate flag if you plan to remove the function at some point (and we may do it if we have general handling for big documents).